src/Lykke.Payments.Link4Pay.DomainServices/Lykke.Payments.Link4Pay.DomainServices.csproj

```diff
@@ -1,8 +1,13 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
```

Comment:
?

Reply:
it's not easy to use .net core 3.1 with current infrastructure packages